src/api/controllers/articleController.js

```diff
@@ -101,4 +102,35 @@ export default class ArticleController {
       message: 'Your Article is up-to-date now, Thanks'
     });
   }
+
+  /**
+   * allow a user to highlight a text in an article
+   *
+   * @author Alain Burindi
+   * @static
+   * @param {object} req the request
+   * @param {object} res the response to be sent
+   * @memberof ArticleController
+   * @returns {Object} res
+   */
+  static async highlight(req, res) {
+    const { articleId } = req.params;
+    const {
+      startIndex, lastIndex, text, comment
+    } = req.body;
+    const correctLength = lastIndex - startIndex + 1;
```

Comment:
I like this idea @alainburindi 
I honestly thought this feature will take more than what is happening here.
`Shout out to you brother`